src/main/resources/ee/ut/timeline.html

```diff
@@ -4,7 +4,6 @@
     <meta charset="UTF-8">
     <title>Title</title>
     <link rel="stylesheet" href="style.css">
-    <script src="data.js"></script>
```

Comment:
Is this really not needed anymore?

Reply:
I think it is not. Preview is working without this. Also when the line is there, I get warning "Cannot resolve file 'data.js'"

Reply:
preview does not use this one